h/api/models/annotation.py

```diff
@@ -103,6 +103,21 @@ def document(self):
         if self.documents:
             return self.documents[0]
 
+    @property
+    def parent_id(self):
+        """
+        Return the ID of the annotation that this annotation is a reply to.
+
+        Returns the ID of the top-level annotation at the top of the thread,
```

Comment:
I don't think this docstring is accurate. The function returns `references[-1]`, which is the immediate ancestor, not the thread root.